cmd/buildah/run.go

```diff
@@ -43,12 +43,14 @@ var (
 	}
 	runDescription = "Runs a specified command using the container's root filesystem as a root\n   filesystem, using configuration settings inherited from the container's\n   image or as specified using previous calls to the config command"
 	runCommand     = cli.Command{
-		Name:        "run",
-		Usage:       "Run a command inside of the container",
-		Description: runDescription,
-		Flags:       runFlags,
-		Action:      runCmd,
-		ArgsUsage:   "CONTAINER-NAME-OR-ID COMMAND [ARGS [...]]",
+		Name:                   "run",
+		Usage:                  "Run a command inside of the container",
+		Description:            runDescription,
+		Flags:                  runFlags,
+		Action:                 runCmd,
+		ArgsUsage:              "CONTAINER-NAME-OR-ID COMMAND [ARGS [...]]",
+		SkipArgReorder:         true,
```

Comment:
should we add these to the other commands?

Reply:
THe skip arg reorder is just for executing commands after the image. So not sure for it.  But the supports short names might be usefull for buildah from/buid, if they support single letter options.